composer.json

```diff
@@ -98,7 +98,7 @@
         "symfony/validator": "^4.3",
         "symfony/yaml": "^4.3",
         "toflar/psr6-symfony-http-cache-store": "^1.1 || ^2.0",
-        "twig/twig": "^1.41 || ^2.0"
+        "twig/twig": "^1.41 || ^2.10"
```

Comment:
this is needed for `filter` (1.41 / 2.10) and apply (1.40 / 2.09)

It doesnt break do anything as its also min requirement of symfony twig bridge 4.3.0:

https://github.com/symfony/twig-bridge/blob/5ed4b37b36e37baeb36028e2e27725c571853444/composer.json#L20